### ISSUE TITLE
Game not overscan safe

### ISSUE THREAD

**Issue by thekovic** [jnmartin84/64doom#25]
CRT TVs would often show the image with a big amount of overscan - that means that edges of the image as rendered would actually show up outside of the TV's screen. For this reason, commercial game developers were advised to keep crucial HUD elements outside of the overscan area. The overscan area can be seen in red on this image (curtesy of the 240p test suite) and it comes out to roughly 10 pixels on each side.

![image](https://github.com/jnmartin84/64doom/assets/72971433/ce0d62ac-cff7-4576-a4ea-aeeec4d408f6)

64doom is currently not overscan safe and playing on a period CRT TV with overscan means some information is outside the visible area.
What would need to be done to account for overscan:
1. Item pickup messages in the top left need to be moved lower and further right.
2. The status bar needs to be moved higher or ideally redesigned entirely. Sources for inspiration include DOOM 64's HUD, Crispy Doom's transparent HUD, or Quake 64's HUD.

I realize that this issue is a combination of low priority and annoying to implement (because of how DOOM's message system works, in particular) but I wanted to list it for completeness.

**Comment by jnmartin84** [comment]
Yeah it has been an annoyance, with the old version using 320x240 I was always willing to ignore the pixels missing on the left and right edges of the screen.

**Comment by jnmartin84** [comment]
I personally am very unlikely to undertake any more major changes or additions, mostly just cleaning up the old things I hacked together, but will keep accepting pull requests as they come in if you want to give it a shot

**Comment by jnmartin84** [comment]
I started developing a 19" RCA CRT back in 2014, I no longer have that screen but even the modern flat panel display I am using still has overscan. I was aware of it, it just didn't bother me enough to deal with beyond trying a few hacks in the past with VI scaling registers.
